packages/user-interface/src/contexts/AppContext.tsx

```diff
@@ -36,15 +39,40 @@ export const AppProvider = ({ children }: MessagesProviderProps) => {
   const location = useLocation();
   const navType = useNavigationType();
   const [firstLoad, setFirstLoad] = useState(true);
+  const { restUri } = useContext(ApiContext);
+  const [logoUrl, setLogoUrl] = useState<string | undefined>(undefined);
   const [themes, setThemes] = useState<Themes>([]);
   const [messagesCount, setMessagesCount] = useState(0);
+  const [loadingConfig, startTransition] = useTransition();
   const { initNavigationItems, updateNavigationItems } =
     useContext(RouterContext);
   const { isLoading: loadingUser } = useContext(UserContext);
   const [history, setHistory] = useState<string[]>(
     JSON.parse(localStorage.getItem("history") || "[]"),
   );
 
+  useEffect(() => {
+    if (window.USE_THEME_API !== "true") return;
+    startTransition(async () => {
+      const response = await fetch(`${restUri}/public/theme/logo`);
+      const base64 = await response.text();
```

Comment:
The response here is already a data URI.

Reply:
I'm saying that the value of `response.text()` is already:
`data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d...`

so the interpolation on the next line wont work. Sorry, my comment was misleading.

Reply:
doing in this way means the implementation can also use the configuraiton property in the backend to refer to a image by url, since the idea is that `logoUrl` can be any src, not just a dataUri
```ts
children: logoUrl ? <img src={logoUrl} alt="logo" /> : <HeaderLogo />,
```